manifests/resource-topology-exporter-ds.yaml

```diff
@@ -53,22 +53,22 @@ spec:
       containers:
       - name: resource-topology-exporter-container
         image: quay.io/swsehgal/resource-topology-exporter:latest
-#        command:
-#        - /usr/local/bin/resource-topology-exporter
-#        - --sleep-interval=1s
         command:
-        - sleep
-        args:
-        - "1000000"
+        - /usr/local/bin/resource-topology-exporter
+        - --watch-namespace=rte
```

Comment:
in a future PR, I believe we should start watching all the namespaces

Reply:
By default all the namespaces are watched, for demo and testing purpose I check for pods in a certain namespace (rte in this case). I can remove --watch-namespace from the yaml if you think that is more appropriate